src/widgets/helper/ResizingTextEdit.cpp

```diff
@@ -270,7 +275,53 @@ bool ResizingTextEdit::canInsertFromMimeData(const QMimeData *source) const
 
 void ResizingTextEdit::insertFromMimeData(const QMimeData *source)
 {
-    if (!source->hasImage())
+    if (source->hasImage())
+    {
+#if 0
+        static QUrl url("http://localhost:7494/upload?password=xd");
+        // default port and password for nuuls' filehost.
+#else
+        static QUrl url("https://i.nuuls.com/upload");
```

Comment:
maybe this can be controlled with an environment variable, like these: https://github.com/Chatterino/chatterino2/blob/master/docs/ENV.md

Reply:
I'm going to do that